source/fuzz/fuzzer_pass_add_stores.cpp

```diff
@@ -117,10 +121,48 @@ void FuzzerPassAddStores::Apply() {
           return;
         }
 
-        // Choose a value at random, and create and apply a storing
-        // transformation based on it and the pointer.
+        bool is_atomic_store = false;
+        uint32_t memory_scope_id = 0;
+        uint32_t memory_semantics_id = 0;
+
+        auto storage_class = GetIRContext()
```

Comment:
It would be cleaner to declare this as SpvStorageClass and static_cast it to that type. (If we didn't do this in fuzzer_pass_add_loads then feel free to do so there in this PR too.)

Reply:
I made it down when passing to the function.
Moved.